packages/contentstack-export-to-csv/src/util/index.js

```diff
@@ -678,6 +683,174 @@ function wait(time) {
   });
 }
 
+function handleErrorMsg(err) {
+  if (err?.errorMessage) {
+    cliux.print(`Error: ${err.errorMessage}`, { color: 'red' });
+  } else if (err?.message) {
+    cliux.print(`Error: ${err.message}`, { color: 'red' });
+  } else {
```

Comment:
Your code going to works fine. Still can you try combain this if possible and that realy makes more readable for other developers?

`cliux.print("Error: ${err?.errorMessage ?? err?.message}", { color: 'red' });`

Reply:
Got it . modified it